README.md

```diff
@@ -27,40 +27,57 @@ A multi-node radio frequency race timing system for FPV drone racing with event
 * Connect other systems to extend functionality via MQTT or JSON API
 * Send realtime lap data to LiveTime
 
-## Improvements
+## Improvements (over RotorHazard 3.0)
 
 * Lap detection based on topological signal processing (persistent homology).
 * Auto-calibration/tuning of nodes:
     * Supervised learning (based on corrected laps).
     * Unsupervised learning (gradient descent).
 * Multi-frequency node support on Arduino Nanos (track more than one frequency per node).
 * MQTT protocol.
-* Frequency band scanner (UI and standalone tool).
-* Support for server (e.g. Raspberry Pi) audio output (e.g. bluetooth speaker).
-* Link pilots with their MultiGP or iFPV accounts.
-* Sync race events with iFPV.
-* BDRA race formats.
+* BDRA race formats, including "Grace period" (eg 30 seconds) after Time.
+* Command line options to specify configuration file and database.
+* GZip as well as Zip encoding
+* WiFi OTA support
+* Race metrics
+* Pluggable Leaderboards
+* Test coverage.
+* Continuous Integration (CI)
+* ...and a massive code clean-up!
+
+## Extensions (over RotorHazard 3.0)
+
+* Support for Arduino-based nodes
```

Comment:
This already existed in 3. Maybe could go in improvements as "Improved I2C handling for Arduino-based nodes".